client/src/components/DogParkShow.js

```diff
@@ -41,10 +43,40 @@ const DogParkShow = (props) => {
         getPark()
     }, [])
 
+    const deletePark = async () => {
+        try {
+            const response = await fetch(`/api/v1/parks/${parkId}`, { method: "DELETE"})
+            if(!response.ok) {
+                const errorMessage = `${response.status} (${response.statusText})`
+                const error = new Error(errorMessage)
+                throw error
+            }
+            setShouldRedirect(true)
+        } catch (err) {
+            console.error(`Error in fetch: ${err.message}`)
+        }
+    }
+
+    const handleOnClickDelete = (event) => {
+        event.preventDefault()
+        deletePark()
+    }
+
+    if(shouldRedirect) {
```

Comment:
there should be a space after `if` before the parentheses
https://github.com/airbnb/javascript#whitespace--around-keywords